Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs

```diff
@@ -27,20 +27,39 @@ public override void OnActivate(string searchContext, VisualElement rootElement)
             BuildUI();
         }
 
+        public override void OnDeactivate()
+        {
+            // Debug.Log("Project settings Editor destroyed, trying to save asset");
+            TrySaveAsset();
```

Comment:
Would there be a need to undo callback registrations from OnDeactivate > BuildUI here when deactivating to avoid additional registrations?

Reply:
Yes 👍